### PR TITLE
fix some typo for thermobox support

### PR DESCRIPTION
**Pull request by shibby20**

### CODE REVIEW

File: blebox_uniapi/box_types.py [hunk 1]
```diff
@@ -149,11 +149,10 @@ def get_latest_api_level(product_type: str) -> Union[dict, int]:
             ],
         }
     },
-    # saunaBox
     "thermoBox": {
         20200229: {
-            "api_path": "/state/extended",
-            "extended_state_path": "/state/extended",
+            "api_path": "/api/thermo/state",
```
Comment (riokuu) [inline blebox_uniapi/box_types.py:154]: Those are deprecated according to:
https://technical.blebox.eu/openapi_thermobox/openAPI_thermoBox_20200229.html#tag/Control-and-State/paths/~1api~1thermo~1state/get

File: blebox_uniapi/box_types.py [hunk 2]
```diff
@@ -170,17 +169,16 @@ def get_latest_api_level(product_type: str) -> Union[dict, int]:
...
-            ]
+            ],
         }
-
     },
     "saunaBox": {
         20180604: {
             # TODO: read extended state only once on startup
-            "api_path": "/api/heat/extended/state",
+            "api_path": "/api/heat/state",
```
Comment (riokuu) [inline blebox_uniapi/box_types.py:181]: Endpoint doesn't provide information about maximumTemp and minimumTemp.
https://technical.blebox.eu/archives/saunaBoxAPI/

**Review by riokuu — COMMENTED** [review]
See comments under each change.

**Comment by shibby20** [comment]
closed

**Pull request closed without merging**